.github/workflows/pull-request.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         java-version: ${{ matrix.java }}
         distribution: 'adopt'
     - name: Run Unit Tests With Maven
-      run: mvn surefire:test --file pom.xml
```

Comment:
This is intentionally only running the unit tests and not the integration tests. Only release candidate PRs run integration tests. All PRs have the unit tests ran.

Reply:
The problem with `mvn surefire:test` is, that compile and test-compile is skipped, which means: there are no tests to run.
See https://github.com/liquibase/liquibase-postgresql/runs/2576578700?check_suite_focus=true#step:5:4310

```
[INFO] --- maven-surefire-plugin:2.12.4:test (default-cli) @ liquibase-postgresql ---
... lots of download message ...
[INFO] No tests to run.
```

Wwith `mvn clean test`, the [maven lifecycle](http://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#Lifecycle_Reference) is executed till the test phase, which runs the unit tests after the code has been compiled. But it doesn't execute the integration tests, which come in a later phase.

Also, running a plugin from command line directly like `mvn surefire:test` uses a different execution configuration (namely `default-cli`) - this could also be a reason, why no tests have been found to run (I don't know, what the defaults there would be for includes...)

So in summary: `mvn clean test` runs the unit tests, `mvn clean verify` runs unit tests+integration tests.
Note I used "verify" here, because sometimes the results of the integration tests are only evaluated in verify (e.g. the failsafe plugin does it that way).